scripts/tooling.py

```diff
@@ -20,26 +20,27 @@ def _run(cmd: str):
 
 
 def format():
-    _run(f"isort --profile=black .")
-    _run("black .")
+    _run("ruff check . --fix")
+    _run("ruff format .")
 
 
-def lint():
-    _run(
-        "flake8 electricitymap tests parsers --count --select=F401,E901,E999,F821,F822,F823 --show-source --statistics"
-    )
-    _run("black --check .")
-    _run("isort -c .")
+def pylint():
```

Comment:
This is crazy slow, do we actually need it or can we remove it?

I looked at the pylint rules added to Ruff so far but they don't seem to be available. Check here: https://docs.astral.sh/ruff/rules/#pylint-pl

Reply:
cc @FelixDQ and @unitrium, think we can remove it now that we use ruff?

Reply:
Yes, we have removed pylint on the internal repo in favor of ruff. The rules don't overlap 100%, but we believe that the rules from ruff are sufficient. So feel free to remove pylint from contrib as well.

Reply:
Thanks! Will do so right away!